JASP-Desktop/po/jasp_nl.po

```diff
@@ -10545,15 +10545,15 @@ msgstr ""
 #, qt-format
 msgctxt "AnalysisForm|"
 msgid ""
-"These components were loaded with a wrong kind of value and are set with "
+"These components were loaded with the wrong type of value and have been reset to "
 "their default values:%1"
 msgstr ""
 "Deze componenten werden geladen met een verkeerd soort waarde en kregen hun "
```

Comment:
Ok, ik durf het bijna niet te zeggen... Maar als je "a" naar "the" veranderd...
Dan moet je natuurlijk ook "een" naar "het" veranderen...

Reply:
Nou... Misschien is dit toch wel goed eigenlijk!